### ISSUE TITLE
Crash when open the storage request

### ISSUE THREAD

**Issue by Wellto0** [badasintended/slotlink#109]
**Setup**
- Minecraft: 1.16.5
- Fabric API: 0.39.2
- slotlink: 2.4.2+1.16.5
- related mod: fabric language kotlin1.6.1+ kotlin 1.5.10

**Problem**
When click to open the storage, the client game crashes, in the "other block" i still can see the chests are linked and working, but cant acess the storage anyway.
Crash report: https://pastebin.com/2wqGnYca

**Reproduction**
Chest with any vanilla item > link clable > cable > blue block > green block > click > crash

**Remarks**
_Additional info lies here_

**Comment by deirn** [comment]
Can you try updating Fabric Language Kotlin and see if it fixes it?

**Comment by Wellto0** [comment]
Yes, I will try it tomorrow, don't saw that has updade for kotlin1.16

**Comment by Wellto0** [comment]
did it and still crashing, in otherway AE2 are working >>dont know if have some simmilarity in code

**Comment by deirn** [comment]
I can't reproduce the crash, can you try deleting `.minecraft/config/slotlink.json` and see if it still crash?

**Comment by Wellto0** [comment]
there's no file with this name in config, 
should be the problem by the way.
The file was not beeing created in config file

**Comment by deirn** [comment]
I really couldn't reproduce the crash :<
Can you try with only Slotlink, Fabric API, and Fabric Language Kotlin and see if it crashes?

**Comment by Wellto0** [comment]
Didn't, but when the with the full modlist, backs again, in the next days i will find the real mod that's crashing dont worry.
